server/index.js

```diff
@@ -390,4 +394,36 @@ app.get("/users", async (req, res) => {
 
 app.listen(port, () => {
   console.log(`You are listening on port${port}`);
-});
\ No newline at end of file
+});
+
+// CHALLENGE FRIEND
+app.get('/email/:friend/:user/:friendEmail/:message', (req, res) => {
```

Comment:
We are going to eventually pass in a link to the quiz

Reply:
I figured. I will definitely add that.